numba/unicode.py

```diff
@@ -622,6 +622,29 @@ def join_str_impl(sep, parts):
         return join_str_impl
 
 
+@overload_method(types.UnicodeType, 'zfill')
+def unicode_zfill(str_to_fill, width_to_fill):
+    def zfill_impl(fstr, fwidth):
+
+        fstr_len = len(fstr)
+
+        if fwidth <= fstr_len or fstr_len == 0:
+            return fstr
+
+        extra_zero = fwidth - fstr_len
+        first_char = fstr[0:1]
+        fill_line = '0' * extra_zero
+
+        if first_char == '+' or first_char == '-':
```

Comment:
Here you can use the common python idiom: `if first_char in ['+', '-']` which saves you a few characters.